### DISCUSSION TITLE
Maximum call stack exeded

### DISCUSSION THREAD

**Discussion by amreeth** [clientIO/joint#2112] [Q&A]
### Introduction

![error 1](https://user-images.githubusercontent.com/89999644/229578551-21b2c3b1-c2cc-4382-8d13-209b20490a88.jpeg)
![img](https://user-images.githubusercontent.com/89999644/229578559-5711362e-90c5-4b19-9bb3-dd2a46352bd4.jpeg)
I connected  multiple links through ui but I am getting this error, how to resolve this error

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment]
You can provide the steps to reproduce (or better a JSFiddle that shows the issue).
